controllers/openstackprovisionserver_controller.go

```diff
@@ -269,6 +248,14 @@ func (r *OpenStackProvisionServerReconciler) Reconcile(ctx context.Context, req
 				break
 			}
 		}
+
+		// Ensure Status.LocalImageURL is up to date
+		localImageURL := r.getLocalImageURL(instance)
+		if localImageURL != instance.Status.LocalImageURL {
+			// Update status with LocalImageURL
+			common.LogForObject(r, fmt.Sprintf("OpenStackProvisionServer LocalImageURL changed: %s", localImageURL), instance)
+		}
```

Comment:
Won't this always fire because L212 will always set `instance.Status.LocalImageURL = ""` whenever we reconcile?

Reply:
yeah, had it working then moved it around... will undo